docs/internals/release-process.txt

```diff
@@ -37,39 +35,84 @@ security purposes, please see :doc:`our security policies <security>`.
 
-    So, for example, if we decided to start the deprecation of a function in
-    Django 1.7:
+    See `the download page`_ for the releases that have been designated for
+    long-term support.
 
-    * Django 1.7 will contain a backwards-compatible replica of the function which
-      will raise a ``RemovedInDjango19Warning``. This warning is silent by
-      default; you can turn on display of these warnings with the ``-Wd`` option
-      of Python.
+    .. _the download page: https://www.djangoproject.com/download/
 
-    * Django 1.8 will still contain the backwards-compatible replica. This
-      warning becomes *loud* by default, and will likely be quite annoying.
+Release cadence
+===============
 
-    * Django 1.9 will remove the feature outright.
+Starting with Django 2.0, version numbers will use a loose form of `semantic
+versioning <http://semver.org/>`_ such that each version following an LTS will
+bump to the next "dot zero" version. For example: 2.0, 2.1, 2.2 (LTS), 3.0,
+3.1, 3.2 (LTS), etc.
 
-  Minor release
-    Minor releases (A.B.C, etc.) will be issued as needed, often to fix security
-    issues.
+SemVer makes it easier to see at a glance how compatible releases are with each
+other. It also helps to anticipate when compatibility shims will be removed.
+It's not a pure form of SemVer as each feature release will continue to have a
+few documented backwards incompatibilities where a deprecation path isn't
+possible or not worth the cost. Also, deprecations started in an LTS release
+(X.2) will be dropped in a non-dot-zero release (Y.1) to accommodate our policy
+of keeping deprecation shims for at least two feature releases. Read on to the
+next section for an example.
 
-    These releases will be 100% compatible with the associated major release,
-    unless this is impossible for security reasons or to prevent data loss.
-    So the answer to "should I upgrade to the latest minor release?" will always
-    be "yes."
+.. _internal-release-deprecation-policy:
+
+Deprecation policy
+==================
+
+A feature release may deprecate certain features from previous releases. If a
+feature is deprecated in feature release X, it will continue to work in feature
+releases X and X+1 but raise warnings. It will usually be removed in feature
+release X+2.
+
+So, for example, if we decided to start the deprecation of a function in
+Django 4.2:
+
+* Django 4.2 will contain a backwards-compatible replica of the function which
+  will raise a ``RemovedInDjango51Warning``. This warning is silent by
+  default; you can turn on display of these warnings with the ``-Wd`` option
+  of Python.
+
+* Django 5.0 (the version that follows 4.2) will still contain the
+  backwards-compatible replica. This warning becomes *loud* by default, and
+  will likely be quite annoying.
+
+* Django 5.1 will remove the feature outright.
+
+The exception to the two release deprecation cycle will be for features
```

Comment:
OK it's explained here, but I see it more the other away around. The main rule is that features are dropped when we increment the major version component and we have an exception for features removed in a LTS so there is still 2 releases to fix your code.